src/passes/pass.cpp

```diff
@@ -267,6 +267,10 @@ void PassRegistry::registerPasses() {
   registerPass("simplify-globals",
                "miscellaneous globals-related optimizations",
                createSimplifyGlobalsPass);
+  registerPass("simplify-globals-optimizing",
+               "miscellaneous globals-related optimizations, and optimizes "
+               "where we replaced global.gets with constants",
```

Comment:
I get the point that this could be clearer, but I'm not sure "locally" is the right term...?

This is the same as with inlining and dae - they also have text like "and optimizes where we removed/inlined/replaced" etc. Maybe we can find something better for all of them?

Reply:
Ah, I thought `optimize` was supposed to control the new linear-execution optimization. This makes more sense now. :+1: